SiraUtil/Tools/FPFC/FPFCToggle.cs

```diff
@@ -180,7 +181,24 @@ private void DisableFPFC()
             foreach (var listener in _fpfcListeners)
                 listener.Disabled();
 
-            InitializeXRLoader();
+            _siraLog.Debug(IsProcessRunning("vrserver") ? "VR Server is running" : "VR Server is not running");
+            _siraLog.Debug(IsProcessRunning("vrcompositor") ? "VR Compositor is running" : "VR Compositor is not running");
+            _siraLog.Debug(Environment.GetCommandLineArgs().Any(s => s.ToLower() == "oculus") ? "Running as vrmode Oculus" : "Running as vrmode Steam");
```

Comment:
The game doesn't make use of `-vrmode oculus` anymore. So that is not reliable to use.